src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -703,17 +700,30 @@
   {% endif %}
 {% endblock birthday_widget %}
 
-{% block generatable_text_widget %}
-  <div class="input-group">
-    {{- block('form_widget') -}}
-    <span class="input-group-btn ml-1">
-      <button class="btn btn-outline-secondary js-generator-btn"
-              type="button"
-              data-target-input-id="{{ id }}"
-              data-generated-value-length="{{ generated_value_length }}"
-      >
-        {{ 'Generate'|trans({}, 'Admin.Actions') }}
-      </button>
-   </span>
+{% block file_widget %}
+  <div class="custom-file">
+    {% set attr = attr|merge({
+      class: (attr.class|default('') ~ ' custom-file-input')|trim,
+      'data-multiple-files-text': '%count% files'|trans({}, 'Admin.Global'),
```

Comment:
Hi @LouiseBonnard 

We have a special case of translation here:

This translation applies pluralization rules. 

![Screenshot(5)](https://user-images.githubusercontent.com/17051250/55222538-d5b8ea80-5214-11e9-9983-abb611e90056.png)

in english its pretty straightforward but for example in Lithuanian language this `%count% files` would be translated like:
`%count% failas | %count% failai | %count% failų`

the pipelines indicate the difference of singular and plurar forms. In Lithuanian there is one singular and two different plural forms. There are many different rules on each language. More info here: https://symfony.com/doc/current/translation.html on pluralization tab.

My only concern is that if people when they see this translation will understand that they should apply these rules?

Reply:
Why is it plural? Couldn't we turn it into a singular form?

Reply:
If I understand you correctly you mean to have the translation like : `%count% file` which will be translated to : `10 file` for instance? :thinking:

Reply:
I don't get it. Is this new file input supposed to be applied to every page migrated in Symfony? And what is the behavior expected?

Reply:
Its new file input which will appear on every migrated PrestaShop page where the files are being uploaded:

so this one 

![Screenshot(7)](https://user-images.githubusercontent.com/17051250/55239639-c3a07180-523f-11e9-8c6e-d7b01b19df25.png)

will become something like this:

![Screenshot(8)](https://user-images.githubusercontent.com/17051250/55239685-de72e600-523f-11e9-8d50-13b157586fee.png)

For this input if you upload multiple files the translation will appear like:

![Screenshot(10)](https://user-images.githubusercontent.com/17051250/55239945-63f69600-5240-11e9-8273-9073820737e1.png)


and this translation `%count% files` is special due to it applies the pluralization rules which for certain languages translation will look something like this: `%count% failas | %count% failai | %count% failų`.

Different forms of `%count% files` are separated by |. I was just woundering if people are aware of that :thinking:

Reply:
cf. my comment below. As for the translation, I don't know... I can ask translators about it but it won't change the fact that we need to write wordings in English. If we can find another way to write it, we can think about it.